modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -4798,25 +4798,25 @@ type Query {
   ): Target
 
   # Target environment for the given observation (or environment id)
-  targetEnvironment(
-    # Observation ID
-    observationId: ObservationId!
-  ): TargetEnvironment
+  # targetEnvironment(
+  #   # Observation ID
+  #   observationId: ObservationId!
+  # ): TargetEnvironment
```

Comment:
Target environment will eventually have guide stars, blind offsets etc.  I suppose we could just get the asterism from there instead of a dedicated `asterism` query if we're trying minimize.

Reply:
Yes, I was thinking you could just query the observation and pull out the target environment that way. As far as I know there isn't a requirement to provide convenience methods for API end-users in XT.